rhizome/common/bin/daemonizer2

```diff
@@ -0,0 +1,66 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require "fileutils"
+require "shellwords"
+require_relative "../lib/util"
+
+if ARGV.count < 2
+  fail "Wrong number of arguments. Expected at least 2, Given #{ARGV.count}"
+end
+
+STATE_MAP = {
+  "dead" => "NotStarted",
+  "running" => "InProgress",
+  "exited" => "Succeeded",
+  "failed" => "Failed"
+}
+
+def get_state(name)
+  state = r("systemctl show -p SubState --value #{name}.service").chomp
+  STATE_MAP[state] || "Unknown"
+end
+
+def clean(name, stdin_path)
+  state = get_state(name)
+  case state
+  when "InProgress", "Unknown"
+    fail "Cannot clean unless the service is in Succeeded or Failed state!"
```

Comment:
This message would need amendment as well given https://github.com/ubicloud/ubicloud/pull/2756/files#r1989969330